trieTools/zeroBalanceSystemAccountChecker/main.go

```diff
@@ -0,0 +1,409 @@
+package main
```

Comment:
this is a large file. Please refactor: split into more components that you can test. Use interfaces.

Reply:
Agree, split file in multiple files